pyproject.toml

```diff
@@ -32,20 +32,23 @@ chgnet = "0.3.8"
 dgl = "2.1.0"
 mace-torch = "0.3.6"
 matgl = "1.1.2"
-numpy = "^1.26.4"
+numpy = "1.26.4"
```

Comment:
Any reason for unpinning these in particular? Particularly with numpy, recommend doing that in a separate PR.

Reply:
You've just enabled 2.0 by unpinning this.

Reply:
reverted